### DISCUSSION TITLE
mjs version

### DISCUSSION THREAD

**Discussion by drBenway** [ing-bank/lion#1528] [Ideas]
I realy like the buildless setup that is described on [modern web](https://modern-web.dev/guides/going-buildless/es-modules/)

Would it be possible to have an mjs version of the components?

**Comment by daKmoR** [comment]
what do you mean with an mjs version?
mjs stands for module javascript and all component code in lion is already es modules?

so I'm not sure what we could offer/change to have a mjs version? 🤔

**Reply by drBenway** [reply]
I think I would be looking for files that I can just drop on a CDN.

For example my-custom-button component is on an Azure CDN and links to the lion-button file. This would make it easy for a companies internal and external devs to use the components without needing a build or going to public npm or internal registries ( Nexus/Artifactory/Gitlab packageregistry )

**Reply by drBenway** [reply]
something like what https://generator.jspm.io provides
